README.md

```diff
@@ -31,10 +31,11 @@ import Ember from 'ember';
 import hbs from 'htmlbars-inline-precompile';
 import connect from 'ember-redux/components/connect';
 import ajax from 'example/utilities/ajax';
+import getUsersByAccountId from '../reducers';
```

Comment:
Oh yeah, I'm all for more docs and keeping the README basic.  I'm thinking the improved documentation can be a separate PR—at which point this can be removed from the README.

On that note, we should probably have an "API" page of the docs, but that's a separate issue.